pkg/defaults/defaults.go

```diff
@@ -112,7 +113,7 @@ func FromConfig(
 	httpClient := retryablehttp.NewClient()
 	httpClient.Logger = nil
 
-	return fromConfig(ctx, config, graphConf, jobSpec, templates, paramFile, promote, client, buildClient, templateClient, podClient, leaseClient, hiveClient, httpClient.StandardClient(), requiredTargets, cloneAuthConfig, pullSecret, pushSecret, api.NewDeferredParameters(nil), censor, consoleHost, nodeName)
+	return fromConfig(ctx, config, graphConf, jobSpec, templates, paramFile, promote, client, buildClient, templateClient, podClient, leaseClient, hiveClient, httpClient.StandardClient(), requiredTargets, cloneAuthConfig, pullSecret, pushSecret, api.NewDeferredParameters(nil), censor, consoleHost, nodeName, targetAdditionalSuffix)
```

Comment:
At this stage adding another parameter doesn't make any difference to this, already totally unreadable, function signature. It's something we have to refactor, not in this PR of course, but we have to.

Reply:
yeah, agreed there is too much going on here...

Reply:
unfortunately this is a common thing in our infra. we can change it as a follow-up.